web/html/static_files.go

```diff
@@ -134,3 +136,35 @@ func (ar *Router) ResetPasswordHandler(pathComponents ...string) http.HandlerFun
 		}
 	}
 }
+
+//LoginHandler handles login page request
```

Comment:
Please, add one space between dashes and comment. Also, end sentences with a period.